webapp/hooks/useL1WalletClient.ts

```diff
@@ -0,0 +1,13 @@
+import { Chain } from 'viem'
+import { useWalletClient } from 'wagmi'
+
+export const useL1WalletClient = function (chainId: Chain['id']) {
```

Comment:
This hook seems to just rename "data" to "l1WalletClient". It may not worth having it at all and just use `useWalletClient` instead.

Reply:
Granted, removed [fedc8e2](https://github.com/hemilabs/ui-monorepo/pull/1125/commits/fedc8e25823bbad804614fd902f1aec32e1f5f51)